web/src/main/java/org/openmrs/web/filter/StartupFilter.java

```diff
@@ -173,11 +173,10 @@ else if (!httpRequest.getServletPath().equals("/" + WebConstants.SETUP_PAGE_URL)
 	private void initializeVelocity() {
 		if (velocityEngine == null) {
 			velocityEngine = new VelocityEngine();
-			
 			Properties props = new Properties();
-			props.setProperty(RuntimeConstants.RUNTIME_LOG, "startup_wizard_vel.log");
+			props.setProperty(RuntimeConstants.RUNTIME_LOG_NAME, "startup_wizard_vel.log");
```

Comment:
Is this change related to updating apache velocity?

Reply:
yes @dkayiwa  the new version of apache velocity 2.0 doesn't have RUNTIME_LOG in its API as compared to the older version of 1.6.2